src/app/util/attribute-storage.cpp

```diff
@@ -1207,6 +1208,8 @@ void emAfLoadAttributeDefaults(EndpointId endpoint, bool ignoreStorage)
     uint16_t attr;
     uint8_t * ptr;
     uint16_t epCount = emberAfEndpointCount();
+    uint8_t attrData[ATTRIBUTE_LARGEST];
+    auto * attrStorage = ignoreStorage ? nullptr : app::GetAttributePersistenceProvider();
```

Comment:
Need to make sure attrStorage is never null if ignoreStorage is false

Reply:
I made the comment because there is a potential path below that will crash if no attribute persistence provider is present (i.e. it's nullptr), but ignoreStorage is false.

Reply:
Addressed in https://github.com/project-chip/connectedhomeip/pull/13600